perma_web/perma/settings/deployments/settings_common.py

```diff
@@ -18,27 +18,50 @@
     "charset": "utf8",
 }
 
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.mysql', # Add 'postgresql_psycopg2', 'mysql', 'sqlite3' or 'oracle'.
-        'NAME': 'perma',                      # Or path to database file if using sqlite3.
-        'USER': 'perma',
-        'PASSWORD': 'perma',
-        'HOST': '',                      # Empty for localhost through domain sockets or '127.0.0.1' for localhost through TCP.
-        'PORT': '3306',                      # Set to empty string for default.
-        'OPTIONS': _mysql_connection_options
+if os.environ.get('DOCKERIZED'):
```

Comment:
Nitpick: do this by modifying the existing dict, e.g. `DATABASES['default']['user'] = 'perma'`? More DRY, easier to see what's different.